content/news/2022-09-02-gsoc-final-blog-david-chocholaty.md

```diff
@@ -0,0 +1,673 @@
+of standards such as  LADSPA, DSSI, LV2, VST2, VST3 and so on,
+in the application instead of offering only the built-in effects or poor API
+for the LV2 standard. After consultation with my mentor, we agreed,
+that I will take on this task as a regular Mixxx contributor after the end
+of GSoC.
+
+
+#### Things I learned from GSoC
+I don't think I can even express how much the GSoC experience has given me.
+Even though I had the experience with open source in one small project,
+the workflow and development for the greater organization as Mixxx
+was completely different and gives me a lot. I have learned so many cool things
+about audio processing, development in C++ with the best practices,
+using the new C++20 standard, testing and benchmarking with the use
+of Google frameworks, and improving my knowledge with git and approaches
+to open source development in general. Thanks to the change
+in the proposed project extension, I learned a lot about real-time audio signal
+processing and about cool data structures. I had a chance to try to design
+the data structure with extensions too. I am really glad about the plan change
+now. I really improved my English, both, written and spoken. I think,
+that it was the best experience so far for me as a developer I ever had.
+It is awesome, that I can publish my work and have immediately the feedback
+and proposed improvements. With that, I liked the open source development
+to just how much I can learn by the awesome people and create new cool stuff.
+Despite I’m a college student, this actually missed me a lot,
+to just have feedback on my work which opens me the opportunities to learn.
+I really felt that I’m a part of the community. I will be happy to continue
+being part of the Mixxx organization and contributing to open source.
+
+#### Conclusion
+The wished new effect was implemented, and the issue which requested
+this new feature was closed with the "Fix Committed" status.  All requirements
+by the Mixxx organization in their project idea, on which the project proposal
+was based, were met. Thanks to enough time in the GSoC Coding period was worked
+on the project extension. Based on the situation and the importance of new
+Mixxx features, the originally proposed extension was replanned and changed.
+The new group delay handling structure was successfully implemented
+and optimized with the implementation of the extended data structure.
+In the GSoC Coding period, the effect was started to optimized and polished
+for minimizing the latency.
+
+During the coding period, I lined up among the top Mixxx contributors
+for the last month, with 61 commits authored, and I became
+the 27th of 238 contributors for the Mixxx application, with 119 commits
+in total.
+
+*August 2022 mixxxdj/mixxx contributors*
+![TODO]({static}/images/news/august-2022-contributors.png)
+
+*Contributor summary*
+![TODO]({static}/images/news/contributor-summary.png)
```

Comment:
Can you redo these screenshots with a dark theme to fit the website better? I think you can find that on github > settings > appearance.